.gitignore

```diff
@@ -1,4 +1,5 @@
 *.pyc
+__javascript__
```

Comment:
Yes, I did that on purpose: currently, having `__javascript__/boolean.js` let's you avoid the hassle of getting a vendored version of Transcrypt and making it work. Later I will remove `__javascript__/boolean.js` and the `.gitignore` will stay.